app/lib/admin_routes.rb

```diff
@@ -0,0 +1,85 @@
+module AdminRoutes
+  def self.extended(router)
+    # rubocop:disable Metrics/BlockLength
+    router.instance_exec do
```

Comment:
well that's an exciting turn of events

Reply:
This is the old hacky way of splitting routes into several files, which we also use in `forem-app`. Once we upgraded to 6.1 we can use the new `draw` DSL method to get this behavior in a more standard way.